business/mesh.go

```diff
@@ -372,6 +374,11 @@ func (in *MeshService) resolveKialiNetwork() (string, error) {
 		// This setup is OK, it's just that it's not within our multi-cluster assumptions.
 		log.Warningf("Cannot resolve the network ID of the cluster where Kiali is hosted: cannot get the sidecar injector config map :%v", err)
 		return "", nil
+	} else if istioSidecarConfig == nil {
+		// Don't return an error, as this may mean that Kiali is not installed along the control plane.
```

Comment:
do you mean not installed along "with" the control plane? (i.e. it is not installed in the control plane namespace?)
```suggestion
		// Don't return an error, as this may mean that Kiali is not installed in the control plane namespace.
```

Reply:
Nope. If I recall correctly, this only to prevent a failure in case Istio is installed as [Primary-Remote](https://istio.io/latest/docs/setup/install/multicluster/primary-remote_multi-network/) (i.e. Istio installed in only one cluster, and controlling multiple clusters), but then somebody chose to install Kiali in one of those cluster where the Istio control plane is absent.